sdk-utils/src/main/kotlin/com/rakuten/tech/mobile/sdkutils/CommonUtil.kt

```diff
@@ -0,0 +1,246 @@
+package com.rakuten.tech.mobile.sdkutils
+
+import android.annotation.SuppressLint
+import android.app.Activity
+import android.content.Context
+import android.content.res.Configuration
+import android.graphics.Color
+import android.os.Build
+import androidx.annotation.ColorInt
+import com.google.android.gms.common.ConnectionResult
+import com.google.android.gms.common.GoogleApiAvailability
+import com.rakuten.tech.mobile.sdkutils.logger.Logger
+import java.io.File
+import java.io.FileOutputStream
+import java.io.IOException
+import java.math.BigInteger
+import java.nio.charset.Charset
+import java.security.MessageDigest
+import java.security.NoSuchAlgorithmException
+import java.text.DateFormat
+import java.text.SimpleDateFormat
+import java.util.*
+
+object CommonUtil {
+    private const val PLAY_SERVICES_RESOLUTION_REQUEST = 9000
+    private val logger = Logger(CommonUtil::class.java.simpleName)
+
+    /**
+     * Get the encoded byte array of string from character set
+     *
+     * @param rawString the raw string.
+     * @param charsetName the character set, i.e. "UTF-8", "UTF-16"
+     * @return the ByteArray of the string
+     */
+    fun getEncodedByteArray(rawString: String, charsetName: String): ByteArray {
+        return rawString.toByteArray(Charset.forName(charsetName))
+    }
+
+    /**
+     * Get the encoded MD5 digest string
+     *
+     * @param rawString the raw string.
+     * @return the MD5 digest string
+     */
+    fun getMD5HashData(rawString: String?): String? {
+        return if (rawString == null) {
+            null
+        } else try {
+            String.format(
+                "%032x",
+                BigInteger(
+                    1,
+                    MessageDigest.getInstance("MD5")
+                        .digest(getEncodedByteArray(rawString, "UTF-8"))
+                )
+            )
+        } catch (swallowed: NoSuchAlgorithmException) {
+            null
+        }
+    }
+
+    /**
+     * Check whether the strings are same.
+     *
+     * @param first the first string
+     * @param second the second string
+     * @return true if same otherwise false
+     */
+    fun isEqual(first: String, second: String): Boolean {
+        return first == second
+    }
+
+    /**
+     * Check the string whether null or empty
+     *
+     * @param rawString the raw string
+     * @return true if null or empty otherwise false
+     */
+    fun isNullOrEmpty(rawString: String?): Boolean {
```

Comment:
No need for this. This is already available for String classes.

Reply:
I also thought this is not required. Removed. Thanks.